tests/integrations/test_db_integrations_sql.py

```diff
@@ -143,6 +145,7 @@ def test_db_user(self):
     @missing_feature(library="nodejs", reason="not implemented yet")
     @missing_feature(library="java_otel", reason="Open Telemetry uses db.name")
     @missing_feature(library="python_otel", reason="Open Telemetry uses db.name")
+    @missing_feature(library="nodejs_otel", reason="Open Telemetry uses db.name")
```

Comment:
It'll use `db.name` forever? if yes, we need to skip it  using `irrelevant` (and probably had a boolean property `open_telemetry` in context, to avoid having dazillion line like this)